### PR TITLE
Update meeting notes

### PR DESCRIPTION
**Pull request by poojaranjan**
Correct 'Summary' section for meeting 81 & 77

### CODE REVIEW

File: All Core Devs Meetings/Meeting 81.md [hunk 1]
```diff
@@ -34,18 +34,18 @@ EIP-1985 | Discussed under `EFI`. Decision required around needing a Hard Fork
...
-**ACTION 81.1**: Ethereum blog to announce officially that ProgPOW is being adopted because of the decision of all core devs.
+**ACTION 81.1**: Zach will share better communication methods to interact with the Open RPC team. 
 
-**ACTION 81.2**: Zach will share better communication methods to interact with the Open RPC team. 
+**ACTION 81.2**: Hudson will merge PR to change the state to Accepted for EIP 778 and EIP 868. 
 
-**ACTION 81.3**: Hudson will merge PR to change the state to Accepted for EIP 778 and EIP 868. 
+## PROPOSALS
 
+**PROPOSAL 81.1**: Berlin would happen when the BLS pre-compile is ready, ProgPOW would happen the next third Wednesday after that.
```
Comment (edsonayllon) [inline All Core Devs Meetings/Meeting 81.md:46]: I believe it's the third Wednesday of the following month. Network Upgrades are targetted for 3rd Wednesdays of the month they are assigned.

Comment (edsonayllon) [reply All Core Devs Meetings/Meeting 81.md:46]: Just making sure people don't interpret it as 3 weeks later, instead of a full month later.

Comment (poojaranjan) [reply All Core Devs Meetings/Meeting 81.md:46]: In the call, the discussion was about 3 weeks. Ref. line no. 434.

Comment (edsonayllon) [reply All Core Devs Meetings/Meeting 81.md:46]: Oh, they were discussing as if it were 3 weeks. 3rd Wednesday in the new EIP centric process means 3rd Wednesday of the month. Or at least that's what Danno and Tim were proposing. https://www.youtube.com/watch?v=vsA57h052hU

**Comment by timbeiko** [comment]
@poojaranjan @edsonayllon I tried merging this and it's not letting me resolve conflicts in the UI. Could either of you try to rebase this off master and re-push? 

Thanks!

**Comment by edsonayllon** [comment]
@poojaranjan would need to address the conflicts. She can do this by clicking fix merge conflicts in this PR. Otherwise, she'd have to merge the latest master into her fork.

**Comment by edsonayllon** [comment]
@poojaranjan 

Do 

```
git fetch upstream
git merger upstream/master
```

**Comment by poojaranjan** [comment]
Created a new PR #275  to address corrections.

**Pull request closed without merging**